core/billing_api.php

```diff
@@ -133,8 +139,8 @@ function billing_get_for_project( $p_project_id, $p_from, $p_to, $p_cost_per_hou
  * @return array The contains billing data grouped by issues, users, and total information.
  * @access public
  */
-function billing_get_summaries( $p_project_id, $p_from, $p_to, $p_cost_per_hour ) {
-	$t_billing_notes = billing_get_for_project( $p_project_id, $p_from, $p_to, $p_cost_per_hour );
+function billing_get_summaries( $p_project_id, $p_from, $p_to, $p_cost_per_hour, $p_subprojects ) {
```

Comment:
The subprojects parameter is missing from phpdoc.  Also I suggest renaming this parameter in general from `subprojects` to `include_subprojects`.

Reply:
I don't know what "missing from phpdoc" means exactly. I searched for "f_bugnote_cost" and "p_cost_per_hour" for any examples and can't find it in other files.

Reply:
I meant the @param was missing from the function documentation.